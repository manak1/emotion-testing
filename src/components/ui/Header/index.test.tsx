import { render } from "~/test/test-utils"
import * as stories from "./index.stories"
import { composeStories } from "@storybook/testing-react"

const { Default } = composeStories(stories)

describe("Headerのテスト", () => {
  it("snapshot", () => {
    const { asFragment } = render(<Default />)
    expect(asFragment()).toMatchSnapshot()
  })
})
