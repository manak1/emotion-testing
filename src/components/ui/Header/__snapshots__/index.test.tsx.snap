// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headerのテスト snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background-color: #fff;
  padding: 8px 0;
  border: 1px solid #d3d3d3;
  max-width: 480px;
  margin: 0 auto;
}

.emotion-1 {
  background-color: #fff;
  position: relative;
  width: 100%;
  max-width: 480px;
  padding: 0 16px;
  margin: 0 auto;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 4px;
}

.emotion-3 {
  width: 33px;
  height: 33px;
  -webkit-transform: scale(-1, 1);
  -moz-transform: scale(-1, 1);
  -ms-transform: scale(-1, 1);
  transform: scale(-1, 1);
}

.emotion-4 {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-5 {
  color: #333;
  font-size: 20px;
  margin: 0;
}

<header
    class="emotion-0"
  >
    <div
      class="emotion-1"
    >
      <div
        class="emotion-2"
      >
        <img
          class="emotion-3"
          src="/logo.svg"
        />
        <a
          class="emotion-4"
          href="/"
        >
          <h1
            class="emotion-5"
          >
            制度作ってみたったー
          </h1>
        </a>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
